__tests__/app.test.js

```diff
@@ -120,3 +129,47 @@ describe("GET/api/articles/article_id", () => {
       });
   });
 });
+
+describe("GET/api/articles", () => {
+  test("returns status:200 and returns all data from all endpoints", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        article.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+        });
+      });
+  });
+  test("returns status:200 and the articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at")
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        expect(article).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("returns status: 400 with invalid sort by query", () => {
+    return request(app)
```

Comment:
This test isn't necessary at this point, the endpoint returns the data sorted by _created_at_ only and doesn't take a query (yet), so this test should be removed